src/software-delivery-machine/blueprint/deploy/localSpringBootDeployOnSuccessStatus.ts

```diff
@@ -14,24 +14,24 @@
  * limitations under the License.
  */
 
-import { logger } from "@atomist/automation-client";
-import { ArtifactDeployer, FunctionalUnit } from "../../../";
-import { retryGoal } from "../../../handlers/commands/RetryGoal";
-import { ArtifactDeploySpec, deployArtifactWithLogs, executeDeployArtifact } from "../../../handlers/events/delivery/deploy/executeDeploy";
+import {logger} from "@atomist/automation-client";
+import {FunctionalUnit} from "../../../blueprint/FunctionalUnit";
+import {retryGoal} from "../../../handlers/commands/RetryGoal";
+import {ArtifactDeploySpec, deployArtifactWithLogs} from "../../../handlers/events/delivery/deploy/executeDeploy";
 import {
     ManagedDeploymentTargeter,
     ManagedDeploymentTargetInfo,
     targetInfoForAllBranches,
 } from "../../../handlers/events/delivery/deploy/local/appManagement";
-import { executableJarDeployer } from "../../../handlers/events/delivery/deploy/local/jar/executableJarDeployer";
-import { StartupInfo } from "../../../handlers/events/delivery/deploy/local/LocalDeployerOptions";
-import { mavenDeployer } from "../../../handlers/events/delivery/deploy/local/maven/mavenSourceDeployer";
-import { ExecuteGoalOnPendingStatus } from "../../../handlers/events/delivery/ExecuteGoalOnPendingStatus";
-import { ExecuteGoalOnSuccessStatus } from "../../../handlers/events/delivery/ExecuteGoalOnSuccessStatus";
-import { StagingDeploymentGoal, StagingEndpointGoal } from "../../../handlers/events/delivery/goals/commonGoals";
-import { OnSupersededStatus } from "../../../handlers/events/delivery/superseded/OnSuperseded";
-import { SourceDeployer } from "../../../spi/deploy/SourceDeployer";
-import { DefaultArtifactStore } from "../artifactStore";
+import {executableJarDeployer} from "../../../handlers/events/delivery/deploy/local/jar/executableJarDeployer";
```

Comment:
Please change IntelliJ settings Editor -> Code -> TypeScript / Within / ES6 Import/Export braces to avoid these distracting changes.

Reply:
I swear I have changed that before. many times.

Reply:
ohhhh i did a different checkout. right. i should have run my program on it!